### ISSUE TITLE
Unable to automatically install 'tzdata2020a' from '.../Artifacts.toml

### ISSUE THREAD

**Issue by oxinabox** [JuliaTime/TimeZones.jl#302]
May be related to #279 but this was not directly an error during download, but after.

```
 Building TimeZones → `~/.julia/packages/TimeZones/tx0tt/deps/build.log`
┌ Error: Error building `TimeZones`: 
│ ERROR: LoadError: Unable to automatically install 'tzdata2020a' from '/root/.julia/packages/TimeZones/tx0tt/Artifacts.toml'
│ Stacktrace:
│  [1] error(::String) at ./error.jl:33
│  [2] #ensure_artifact_installed#42(::Pkg.BinaryPlatforms.Platform, ::Bool, ::typeof(Pkg.Artifacts.ensure_artifact_installed), ::String, ::Dict{String,Any}, ::String) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.3/Pkg/src/Artifacts.jl:873
│  [3] (::Pkg.Artifacts.var"#kw##ensure_artifact_installed")(::NamedTuple{(:platform,),Tuple{Pkg.BinaryPlatforms.Linux}}, ::typeof(Pkg.Artifacts.ensure_artifact_installed), ::String, ::Dict{String,Any}, ::String) at ./none:0
│  [4] do_artifact_str(::String, ::Dict{String,Any}, ::String, ::Module) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.3/Pkg/src/Artifacts.jl:983
│  [5] #invokelatest#1 at ./essentials.jl:709 [inlined]
│  [6] invokelatest at ./essentials.jl:708 [inlined]
│  [7] macro expansion at /root/.julia/packages/TimeZones/tx0tt/src/compat.jl:42 [inlined]
│  [8] #build#36(::Bool, ::typeof(TimeZones.TZData.build), ::String, ::Array{String,1}, ::String, ::String, ::String) at /root/.julia/packages/TimeZones/tx0tt/src/tzdata/build.jl:52
│  [9] #build at ./none:0 [inlined]
│  [10] build(::String) at /root/.julia/packages/TimeZones/tx0tt/src/tzdata/build.jl:113
│  [11] #build#15(::Bool, ::typeof(build), ::String) at /root/.julia/packages/TimeZones/tx0tt/src/build.jl:11
│  [12] build at /root/.julia/packages/TimeZones/tx0tt/src/build.jl:11 [inlined] (repeats 2 times)
│  [13] top-level scope at /root/.julia/packages/TimeZones/tx0tt/deps/build.jl:3
│  [14] include at ./boot.jl:328 [inlined]
│  [15] include_relative(::Module, ::String) at ./loading.jl:1105
│  [16] include(::Module, ::String) at ./Base.jl:31
│  [17] include(::String) at ./client.jl:424
│  [18] top-level scope at none:5
│ in expression starting at /root/.julia/packages/TimeZones/tx0tt/deps/build.jl:3
└ @ Pkg.Operations /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.3/Pkg/src/backwards_compatible_isolation.jl:649
```

this occured during nightly CI for one of our internal packages

**Comment by kdheepak** [comment]
For what it is worth, I'm also having the same issue on my local machine, but only with the Julia nightly. It works fine with the latest stable release: Julia-1.5.2.

**Comment by omus** [comment]
Hopefully this was just an issue with Julia nightly. Just to check have you see any of these errors since?

**Comment by oxinabox** [comment]
I have not

**Comment by omus** [comment]
In PR https://github.com/JuliaTime/TimeZones.jl/pull/441 the `deps/build.jl` was removed so this is no longer an issue. Included in TimeZones release 1.12
